api/v1/loadtest_types.go

```diff
@@ -21,7 +21,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// NOTE: AFTER EDITS, YOU MUST RUN `make` TO REGENERATE CODE.
+// NOTE: CODE IS GENERATED FROM THIS FILE!
```

Comment:
I would keep the comment more or less as before, except for `make manifests` AND `make` insted of just `make`.

Reply:
Should be fixed. What do you think?